src/NosCore.GameObject/Services/Inventory/InventoryService.cs

```diff
@@ -248,8 +248,8 @@ public IItemInstance MoveInPocket(short sourceSlot, PocketType sourceType, Pocke
 
             switch (sourceInstance?.Item.ItemType)
             {
-                case ItemType.Fashion when targetType != PocketType.Main && targetType != PocketType.Costume:
-                case ItemType.Specialist when targetType != PocketType.Main && targetType != PocketType.Specialist:
+                case ItemType.Fashion when targetType != PocketType.Equipment && targetType != PocketType.Costume:
```

Comment:
in fact this code is useless we should move it to a specific place just over we should handle 
not ItemType.Fashion || Sp when targetType == PocketType.Sp || targetType == PocketType.Costume